Makefile

```diff
@@ -35,30 +70,30 @@ release: \
 # matchbox
 
 bin/linux-amd64/matchbox:
```

Comment:
But it was kinda clear before right? I'm for Makefiles, but some of the features one might use are so opaque